domenetjenester/beregningsgrunnlag/src/main/java/no/nav/foreldrepenger/domene/migrering/BeregningMigreringTjeneste.java

```diff
@@ -149,6 +156,18 @@ private void migrerBehandling(Behandling behandling) {
                 oppdaterKoblingMedStpGrunnbeløpOgReguleringsbehov(kobling, response.grunnlag().getBeregningsgrunnlag());
             }
 
+            Optional<LocalDate> førsteUttaksdato;
+            try {
+                førsteUttaksdato = skjæringstidspunktTjeneste.getSkjæringstidspunkter(ref.behandlingId()).getFørsteUttaksdatoHvisFinnes();
+            } catch (Exception e) {
+                LOG.info("Fant ikke første uttaksdato for behandling {}, setter den til LocalDate.MIN", ref.behandlingUuid());
+                førsteUttaksdato = Optional.of(LocalDate.MIN);
+            }
+            // Må ha false toggle her til fpkalkulus er prodsatt
+            if (false && førsteUttaksdato.map(this::kanPåvirkesAvÅretsGregulering).orElse(false)) {
```

Comment:
Ble mer robust nå. 
Lite tips - jeg skrev også map/boolean/orElse-false før.
Mer Stream-orientert er førsteUttaksdato.filter(this::kanPåvirkesAvÅretsGregulering).isPresent()

Reply:
Godt poeng, kan endre til det